.env.sample

```
@@ -6,4 +6,5 @@ DB_USER=
 DB_PASSWORD=
 DB_HOST=
 DB_PORT=
+token.json
```

```suggestion
token.json
```
這裡的 `token.json` 要加到 .env 裡面嗎？